src/main/java/edu/cornell/kfs/module/purap/service/impl/CuElectronicInvoiceMatchingServiceImpl.java

```diff
@@ -329,11 +329,12 @@ private boolean eInvoiceVendorMatchesPOVendor(PurchaseOrderDocument poDoc, Elect
         VendorDetail vendorByDunsDetail = vendorService.getVendorByDunsNumber(orderHolder.getElectronicInvoice().getDunsNumber());
 
         if (poDoc.getVendorHeaderGeneratedIdentifier() == null || poDoc.getVendorDetailAssignedIdentifier() == null
-                || ObjectUtils.isNull(vendorByDunsDetail) || !(poDoc.getVendorHeaderGeneratedIdentifier()
-                        .equals(vendorByDunsDetail.getVendorHeaderGeneratedIdentifier()))) {
+                || ObjectUtils.isNull(vendorByDunsDetail))
+        {

```

Comment:
Did you mean for the opening brace to be on it's own line?

Reply:
I did not, fixing that now. Thank you for niticing